templates/flasher.lua

```diff
@@ -1565,4 +1566,36 @@ function M.identify(tPlugin, aAttr, fnCallbackProgress, fnCallbackMessage)
 	return ulValue == 0
 end
 
+
+--------------------------------------------------------------------------
+-- Function to reset a netX through the flasher binary by
+-- triggering a watchdog reset
+--------------------------------------------------------------------------
+function M.reset(tPlugin, aAttr, fnCallbackProgress, fnCallbackMessage)
+	local romloader = require("romloader")
```

Comment:
We shouldn't need this line as the romloader is already loaded at the beginning of the script.

Reply:
Correct, removed it.